### PR TITLE
apply -> register

### PR DESCRIPTION
**Pull request by adamdruppe**

### CODE REVIEW

File: app/views/enrollments/_form.html.erb [hunk 1]
```diff
@@ -14,7 +14,7 @@
           </header>
 
           <p class="help coach volunteer">
-           To apply to volunteer as a <%= @program_title %> Leadership Coach in our <%= @program_site %> program, please complete and submit this application in its entirety. Your answers will be automatically saved as you move between questions, and you can safely leave this page and log back in at any time to resume your work. Your application won't be considered for evaluation until you click SEND at the bottom of the form.
+           To register to volunteer as a <%= @program_title %> Leadership Coach in our <%= @program_site %> program, please complete and submit this form in its entirety. Your answers will be automatically saved as you move between questions, and you can safely leave this page and log back in at any time to resume your work. Your registration won't be processed until you click SEND at the bottom of the form.
```
Comment (sadleb) [inline app/views/enrollments/_form.html.erb]: I think we only want this change to affect non-LC volunteers, not coaches.  Or did Sheila suggest we move to register language with coaches as well?

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -138,7 +138,14 @@
                   <button type="submit" class="button-primary">
                     <!-- We check the model campaign id to see if they will be auto-mapped based on query params. If so, next
                          step is start application, so it says that. Otherwise, generic "submit" is used. -->
-                    <%= (@user.applicant_type == 'volunteer' || @user.applicant_type == 'undergrad_student' || @user.applicant_type == 'temp_volunteer') ? "Next: Start Application" : "Submit" %></button>
+                    <% if(@user.applicant_type == 'volunteer' || @user.applicant_type == 'undergrad_student') %>
```
Comment (sadleb) [inline app/views/users/new.html.erb]: Should this be ```temp_volunteer``` ?

File: app/views/enrollments/_form.html.erb (hunk 1 above)
Comment (adamdruppe) [inline app/views/enrollments/_form.html.erb]: On Tue, Feb 16, 2016 at 10:07:41AM -0800, Brian Sadler wrote:
> I think we only want this change to affect non-LC volunteers, not coaches.  Or did Sheila suggest we move to register language with coaches as well?

Oh yeah, I didn't realize that was a shared block.

File: app/views/enrollments/_form.html.erb [hunk 3]
```diff
@@ -13,8 +13,12 @@
           <h1>Application for <%= @program_title %> - <%= @program_site %><br /><%= Date.today.year %></h1>
           </header>
 
-          <p class="help coach volunteer">
-           To apply to volunteer as a <%= @program_title %> Leadership Coach in our <%= @program_site %> program, please complete and submit this application in its entirety. Your answers will be automatically saved as you move between questions, and you can safely leave this page and log back in at any time to resume your work. Your application won't be considered for evaluation until you click SEND at the bottom of the form.
+          <p class="help coach">
+           To register to volunteer as a <%= @program_title %> Leadership Coach in our <%= @program_site %> program, please complete and submit this application in its entirety. Your answers will be automatically saved as you move between questions, and you can safely leave this page and log back in at any time to resume your work. Your application won't be considered for evaluation until you click SEND at the bottom of the form.
```
Comment (sadleb) [inline app/views/enrollments/_form.html.erb]: This still says register when it used to say Apply....

**Pull request merged**